plugins/linux/ifplugin/descriptor/interface_loop.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2019 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package descriptor
+
+import (
+	interfaces "github.com/ligato/vpp-agent/api/models/linux/interfaces"
+	"github.com/ligato/vpp-agent/plugins/linux/ifplugin/ifaceidx"
+
+	nslinuxcalls "github.com/ligato/vpp-agent/plugins/linux/nsplugin/linuxcalls"
+	"strings"
+)
+
+// createLoopback adds logical name as alias to linux loopback interface
+func (d *InterfaceDescriptor) createLoopback(nsCtx nslinuxcalls.NamespaceMgmtCtx,
+	linuxIf *interfaces.Interface) (metadata *ifaceidx.LinuxIfMetadata, err error) {
+
+	hostName := getHostIfName(linuxIf)
+	agentPrefix := d.serviceLabel.GetAgentPrefix()
+
+	revert, err := d.nsPlugin.SwitchToNamespace(nsCtx, linuxIf.Namespace)
+	if err != nil {
+		d.log.Error(err)
+		return nil, err
+	}
+	defer revert()
+
+	// add alias in order to include loopback in retrieve output
+	err = d.ifHandler.SetInterfaceAlias(hostName, agentPrefix+linuxIf.Name)
+	if err != nil {
+		d.log.Error(err)
+		return nil, err
+	}
+
+	link, err := d.ifHandler.GetLinkByName(hostName)
+	if err != nil {
+		d.log.Error(err)
+		return nil, err
+	}
+
+	metadata = &ifaceidx.LinuxIfMetadata{
+		Namespace:    linuxIf.Namespace,
+		LinuxIfIndex: link.Attrs().Index,
+	}
+
+	return metadata, nil
+}
+
+// deleteLoopback clear associated interface alias
+func (d *InterfaceDescriptor) deleteLoopback(nsCtx nslinuxcalls.NamespaceMgmtCtx, linuxIf *interfaces.Interface) error {
+	hostName := getHostIfName(linuxIf)
+
+	revert, err := d.nsPlugin.SwitchToNamespace(nsCtx, linuxIf.Namespace)
```

Comment:
This step can be omitted if the loopback is in default namespace if it is a valid case (also for delete)